operator/internal/manifests/internal/config/loki-runtime-config.yaml

```
@@ -73,6 +73,46 @@ overrides:
     {{- end }}
     {{- end}}
   {{- end -}}
+  {{- if $l := $spec.OTLP }}
+    otlp_config:
+      {{- with $l.ResourceAttributes }}
+      resource_attributes:
+        ignore_defaults: {{ .IgnoreDefaults }}
+        {{- with .Attributes}}
+        attributes_config:
+        {{- range . }}
+        - action: {{ .Action }}
+          attributes:
+          {{- range .Attributes }}
+          - {{ . }}
+          {{- end }}
+          regex: {{ .Regex }}
```

See comment in `loki-config.yaml`:

```suggestion
        - action: {{ .Action }}
          {{- with .Regex }}
          regex: {{ . }}
          {{- end }}
          attributes:
          {{- range .Attributes }}
          - {{ . }}
          {{- end }}
```